audit-service/src/main/java/uk/gov/gchq/palisade/service/audit/stream/SerDesConfig.java

```diff
@@ -0,0 +1,146 @@
+     * Used by the Rest Controller to insert requests onto the topic
+     *
+     * @return an appropriate key serialiser for the topic's message content
+     */
+    public static Serializer<String> errorKeySerializer() {
+        return new StringSerializer();
+    }
+
+    /**
+     * Kafka value serialiser for upstream messages coming in as input
+     * Used by the Rest Controller to insert requests onto the topic
+     *
+     * @return an appropriate value serialiser for the topic's message content (AuditErrorMessage)
+     */
+    public static Serializer<AuditErrorMessage> errorValueSerializer() {
+        return (String ignored, AuditErrorMessage auditRequest) -> {
+            try {
+                return MAPPER.writeValueAsBytes(auditRequest);
+            } catch (IOException e) {
+                throw new SerializationFailedException(SERIALIZATION_FAILED_MESSAGE + auditRequest.toString(), e);
+            }
+        };
+    }
+
+    /**
+     * Kafka key deserialiser for upstream messages coming in as input
+     *
+     * @return an appropriate key deserialiser for the topic's message content
+     */
+    public static Deserializer<String> errorKeyDeserializer() {
+        return new StringDeserializer();
+    }
+
+    /**
+     * Kafka value deserialiser for upstream messages coming in as input
+     *
+     * @return an appropriate value deserialiser for the topic's message content (AuditErrorMessage)
+     */
+    public static Deserializer<AuditErrorMessage> errorValueDeserializer() {
+        return (String ignored, byte[] auditRequest) -> {
+            try {
+                return MAPPER.readValue(auditRequest, AuditErrorMessage.class);
+            } catch (IOException e) {
+                throw new SerializationFailedException(DESERIALIZATION_FAILED_MESSAGE + new String(auditRequest, Charset.defaultCharset()), e);
+            }
+        };
+    }
+
+    /**
+     * Kafka key serialiser for upstream messages coming in as input
+     * Used by the Rest Controller to insert requests onto the topic
+     *
+     * @return an appropriate key serialiser for the topic's message content
+     */
+    public static Serializer<String> successKeySerializer() {
+        return new StringSerializer();
+    }
+
+    /**
+     * Kafka value serialiser for upstream messages coming in as input
+     * Used by the Rest Controller to insert requests onto the topic
+     *
+     * @return an appropriate value serialiser for the topic's message content (AuditSuccessMessage)
+     */
+    public static Serializer<AuditSuccessMessage> successValueSerializer() {
+        return (String ignored, AuditSuccessMessage auditRequest) -> {
+            try {
+                return MAPPER.writeValueAsBytes(auditRequest);
+            } catch (IOException e) {
+                throw new SerializationFailedException(SERIALIZATION_FAILED_MESSAGE + auditRequest.toString(), e);
+            }
+        };
+    }
```

Comment:
The Audit Service doesnt seralize anything does it? so these shouldnt be required. Go through the akka config files and see if theres anything you can remove, like producer beans/configs

Reply:
I think it does, the [test] rest endpoint requires this does it not?

Reply:
Yep, the serialisation is used for the requestSink beans that are needed for the endpoint tests